_docs/howto/redaxo_4_5_upgrade.md

```diff
@@ -50,9 +50,36 @@ Als Lösung und Ersatz des Multiuploader-AddOn stehen die AddOns ***uploader***
 ### Redaxo 4 Passwörter migrieren
 **WICHTIG: rex_user Tabelle zu Beginn sichern!** Redaxo 4 Passwörter können mit folgendem Snippet Redaxo 5 kompatibel gemacht werden. 
 
+#### To-Do: Tabelle
 Folgende Schritte sind notwendig um alte Redaxo 4 Passwörter auch in Redaxo 5 verwenden zu können.
```

Comment:
Bitte Wörter mit Bindestrich "durchkoppeln" und kein Leerzeichen in Komposita.

Zu Deutsch:  `REDAXO-4-Passwörter` statt `REDAXO 4 Passwörter`. Würde den Text aber entsprechend meines Vorschlags allgemeiner halten und von "z.B. Passwörter aus REDAXO 4` schreiben, denn die Vorgehensweise zum Hashen gilt auch für xcom -> ycom